packages/discord.js/src/structures/interfaces/TextBasedChannel.js

```diff
@@ -81,7 +81,7 @@ class TextBasedChannel {
    * @property {ReplyOptions} [reply] The options for replying to a message
    * @property {StickerResolvable[]} [stickers=[]] The stickers to send in the message
    * @property {MessageFlags} [flags] Which flags to set for the message.
-   * <info>Only `MessageFlags.SuppressEmbeds` can be set.</info>
+   * <info>Only `MessageFlags.SuppressEmbeds` and `MessageFlags.SuppressNotifications` can be set.</info>
```

Comment:
Why don't we just document it? `* @property {MessageFlags.SuppressEmbeds|MessageFlags.SuppressNotifications} [flags] Which flags to set for the message.`, for example?

Reply:
Shouldn't it be `BitFieldResolvable`?

Reply:
1. MessageFlags isnt a thing
2. maybe make a `MessageFlagsResolvable` definition?

Reply:
`MessageFlags` definitely exists: https://discord-api-types.dev/api/discord-api-types-v10/enum/MessageFlags

Reply:
What should I choose and is this even related to this PR?